include/sys/mem_manage.h

```diff
@@ -9,6 +9,9 @@
 
 #include <sys/util.h>
 #include <toolchain.h>
+#if defined(CONFIG_ARM_MMU) || defined(CONFIG_ARM_MPU)
```

Comment:
Not sure, can we at least add a `CONFIG_ARM64`,  because both aarch32 and aarch64 have `CONFIG_ARM_MPU`.

Reply:
Not a big problem (macros don't affect anything), but IMHO, it's better to add the CONFIG_ARM64 since the header belongs to arm64

Reply:
yes, the definition is for arm64 and also with MMU, so updated the patch, thanks.